app/controllers/books_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to @book, notice: 'Book was successfully created.' }
+        format.html { redirect_to @book, notice: (t 'message.create_info') }
```

Comment:
```suggestion
        format.html { redirect_to @book, notice: t('message.create_info') }
```

というスタイルで書くことの方が多いと思いますー。